### PR TITLE
Ap/tsar test data

### PR DESCRIPTION
**Pull request by austinhpatton**
Updated tsar test data

### CODE REVIEW

File: phylorthology/tsar_downsamp_test_parameters.csv [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "input": "https://github.com/Arcadia-Science/tsar-phylorthology/blob/main/samplesheets/downsamp_test_tsar_samplesheet.csv",
+  "outdir": "results-downsamp-tsar-test",
+  "mcl_inflation": "1.0,2.0,3.0,4.0,5.0",
+  "min_num_spp_per_og": "4",
```
Comment (mertcelebi) [inline phylorthology/tsar_downsamp_test_parameters.csv]: These should be numbers not strings I think? Either the pipeline defaults are wrong or these are wrong.

File: phylorthology/downsamp_test_tsar_samplesheet.csv [hunk 2]
```diff
@@ -0,0 +1,7 @@
+species,file,taxonomy,shallow_db,broad_db,mode,uniprot,mcl_test
+Tetrahymena_thermophila,/home/ubuntu/environment/github/test-datasets/phylorthology/tsar/downsampled_proteomes/Tetrahymena_thermophila.fasta,Alveolata,alveolata_odb10,eukaryota_odb10,proteins,true,false
```
Comment (mertcelebi) [inline phylorthology/downsamp_test_tsar_samplesheet.csv]: These should reference the GitHub paths similar to samplesheet_test.csv

**Review by mertcelebi — CHANGES_REQUESTED** [review]

File: phylorthology/tsar_downsamp_test_parameters.csv (hunk 1 above)
Comment (austinhpatton) [reply phylorthology/tsar_downsamp_test_parameters.csv]: Nope, you're right

File: phylorthology/downsamp_test_tsar_samplesheet.csv (hunk 2 above)
Comment (austinhpatton) [reply phylorthology/downsamp_test_tsar_samplesheet.csv]: yupyup

File: phylorthology/tsar_downsamp_test_parameters.csv (hunk 1 above)
Comment (austinhpatton) [reply phylorthology/tsar_downsamp_test_parameters.csv]: Fixed that, and also simplified the tree inference (i.e. no pmsf tree model)

File: phylorthology/downsamp_test_tsar_samplesheet.csv [hunk 3]
```diff
@@ -0,0 +1,7 @@
+species,file,taxonomy,shallow_db,broad_db,mode,uniprot,mcl_test
+Tetrahymena_thermophila,https://github.com/Arcadia-Science/test-datasets/blob/ap/tsar-test-data/phylorthology/Tetrahymena_thermophila.fasta,Alveolata,alveolata_odb10,eukaryota_odb10,proteins,true,false
```
Comment (mertcelebi) [inline phylorthology/downsamp_test_tsar_samplesheet.csv]: Just reference `main` on these

**Review by mertcelebi — CHANGES_REQUESTED** [review]
a quick fix

File: phylorthology/downsamp_test_tsar_samplesheet.csv (hunk 3 above)
Comment (austinhpatton) [reply phylorthology/downsamp_test_tsar_samplesheet.csv]: Ah, yup

**Review by mertcelebi — APPROVED** [review]

**Pull request merged**